lib/ex_fizz_buzz_enterprise/input_worker.ex

```diff
@@ -22,13 +21,24 @@ defmodule ExFizzBuzzEnterprise.InputWorker do
     GenServer.cast(__MODULE__, :next)
```

Comment:
Another bit of style, if you have an API that wraps `GenServer.call`/cast/etc., use consistent names between function calls and message semantics.  So maybe this function should be `def next`.